recipes/biogeme-optimization/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "biogeme-optimization" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme_optimization-{{ version }}.tar.gz
+  sha256: a4e964d034208eb762217556112369d9f8f7dfeabe603a2dde360bca50031fca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.24.2
+    - scipy >=1.10.1
+    - matplotlib-base >=3.8.2
```

Comment:
These are the dependencies in the `pyproject.toml` file in biogeme-optimization 0.0.10:

```
    "numpy>=2.0.0, <3",
    "scipy>=1.14.0, <2",
    "matplotlib>=3.9.0, <4",
    "tomlkit>=0.12.5, <1"
```

This NumPy >= 2 requirement is going to make it impossible to install in the base environment in this year's TU Delft distribution because we are shipping NumPy 1. I'm not sure why biogeme-optimization requires NumPy 2 now. I raised this issue here: https://github.com/michelbierlaire/cythonbiogeme/issues/21 In the cythonbiogeme-feedstock I just ignored the NumPy 2.0 runtime requirement because if you build against the ABI of NumPy 2 it should work with a NumPy 1 when run. We could try that here also, maybe it runs with NumPy 1 and the requirement is unnecessarily strict.

Reply:
So I should try here with NumPy >= 2 if I understood correctly?

Reply:
I would try leaving it off.